### ISSUE TITLE
Microsoft Arc Mouse scroll wheel stops working after idle until bluetooth service restart

### ISSUE THREAD

**Issue by draeder** [feren-OS/Bug-Reporting-Center#78]
Microsoft Arc Mouse scroll wheel stops working after idle until bluetooth service restart

**Comment by feren** [comment]
Is this mouse using a USB Connector to receive input signals or is it entirely wireless bluetooth?

**Comment by draeder** [comment]
Yes, I'm using an ASUS USB dongle. I've attempted updating Bluez manually and also applied the Bluez update you pushed out last night to no avail. I think it may be an upstream bug in the Ubuntu kernel. There are lots of reports of it in the wild. Seems to have been an issue since at latest 18.04 affecting many people. 

I followed some steps that have minimized it to only happening after idling for *some period* of time (it used to happen every 30 minutes even if actively using the computer)--but I'm afraid I no longer have the link to those steps so I can't relay them here. It's extraordinarily annoying, but not too hard to work around.

**Comment by feren** [comment]
Feren OS uses Kernel 5.0 / Ubuntu 18.04 HWE Stack so it might indeed be upstream in the kernel.

**Comment by feren** [comment]
A good way to check is to stick something like Linux Mint 19.3 on a USB Stick and test in a Live Session of Mint 19.3 or another OS with Linux 5.0 or newer to see if the problem also occurs there.

**Comment by draeder** [comment]
@feren I will give this a try when I have some more time and update with the results. The main problem I have in testing is I don't know exactly when the problem happens. I only know that it's after computer idle for X time. And I've already made some changes based on answers I found on StackExchange that helped. Prior to those changes, the issue would happen even while the computer was actively being used. After those changes, the issue now only happens on idle. 

Either way, I'll let you know what happens. I wish I had another bluetooth mouse to test... But I believe this might specific to the Microsoft Arc Mouse.
